src/main/java/org/jvnet/hudson/update_center/Main.java

```diff
@@ -87,6 +93,15 @@
     @Option(name="-download",usage="Build mirrors.jenkins-ci.org layout")
     public File download = null;
 
+    @Option(name="-cache",usage="Cache target plugin")
+    public File cache = null;
+
+    @Option(name="-cacheAll",usage="Cache all version of plugin")
```

Comment:
Nit: The majority of existing options use `-foo-bar` format for parameters, so please use that instead.